docs/routes.json

```
@@ -130,6 +130,10 @@
             "path": "/guides/schema-validation/syntax"
           }
         ]
+      },
+      {
+        "title": "Liveblocks 1.0 Pricing Model",
```

```suggestion
        "title": "Pricing",
```

Simplified, but don't forget we don't use Camel Case